src/parser/types.ts

```diff
@@ -532,7 +676,26 @@ export interface IndividualComponentSingleResponseCondition {
   to: string;
 }
 
-/** The IndividualComponentAllResponsesCondition interface is used to define a SkipCondition based on all answers to a specific component. If the component is repeated within the block, this condition will only check the first instance of the component once the order is flattened. If you need to check all instances of a repeated component, you should use the RepeatedComponentBlockCondition. */
+/** The IndividualComponentAllResponsesCondition interface is used to define a SkipCondition based on all answers to a specific component. If the component is repeated within the block, this condition will only check the first instance of the component once the order is flattened. If you need to check all instances of a repeated component, you should use the RepeatedComponentBlockCondition.
+ *
```

Comment:
Same as above -- consider using an info admonition for the "If the component is repeated within the block..."

Reply:
These are actually wrong. Fixed them up in aa45ce2